envoyauth/response.go

```diff
@@ -280,6 +282,33 @@ func (result *EvalResult) GetResponseHTTPStatus() (int, error) {
 	return http.StatusForbidden, result.invalidDecisionErr()
 }
 
+// GetDynamicMetadata returns the dynamic metadata to return if part of the decision
+func (result *EvalResult) GetDynamicMetadata() (*_structpb.Struct, error) {
+	var (
+		val interface{}
+		ok  bool
+	)
+	switch decision := result.Decision.(type) {
+	case bool:
+		if decision {
+			return nil, fmt.Errorf("dynamic metadata undefined for simple 'allow'")
```

Comment:
What about `dynamic metadata undefined for simple 'allow'` -> `dynamic metadata undefined for boolean decision`

Reply:
done!